test/addTarget.js

```diff
@@ -18,6 +18,7 @@
 var fullProject = require('./fixtures/full-project')
     fullProjectStr = JSON.stringify(fullProject),
     pbx = require('../lib/pbxProject'),
+    pbxFile = require('../lib/pbxFile'),
```

Comment:
Would you mind if we move this fix into its own PR?

Reply:
Sure. For something like this do you prefer an issue raised, and then a PR resolving the issue? Or just a PR with a description?

Reply:
> do you prefer an issue raised

not needed

> just a PR with a description

That would be great, thanks!

Reply:
https://github.com/apache/cordova-node-xcode/pull/67 - thanks.